rrule/src/iter/rrule_iter.rs

```diff
@@ -58,34 +88,30 @@ impl<'a> RRuleIter<'a> {
         if let Some(err) = self.get_err() {
```

Comment:
Then, I think we can remove the clippy attribute and TODO

Reply:
Agreed